riotfile.py

```diff
@@ -66,7 +67,7 @@ def select_pys(min_version=min(SUPPORTED_PYTHON_VERSIONS), max_version=max(SUPPO
         Venv(name="tracer", command="pytest tests/tracer/", venvs=[Venv(pys=select_pys(), pkgs={"msgpack": latest})]),
```

Comment:
Delete this:

```suggestion
```

Reply:
i swear i had done that in the merge resolve 😕. Good catch!